app/src/main/java/com/lastaoutdoor/lasta/ui/screen/activities/ActivityScreen.kt

```diff
@@ -0,0 +1,247 @@
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.lastaoutdoor.lasta.R
+import com.lastaoutdoor.lasta.data.model.activity.OutdoorActivity
+
+@Composable
+fun ActivityScreen(activity: OutdoorActivity) {
+  LazyColumn(modifier = Modifier.padding(8.dp)) {
+    item { Spacer(modifier = Modifier.height(15.dp)) }
+    // contains the top icon buttons
+    item { TopBar() }
+    // displays activity title and duration
+    item { ActivityTitleZone(activity = activity) }
+    // displays activity difficulty, ration and view on map button
+    item { MiddleZone(activity) }
+    // filled with a spacer for the moment but will contain address + community
+    item { Spacer(modifier = Modifier.height(350.dp)) }
+    // Bottom start activity button
+    item { StartButton() }
+  }
+}
+
+@Composable
+fun StartButton() {
+  Row(
+      modifier = Modifier.fillMaxWidth().testTag("Start button"),
+      horizontalArrangement = Arrangement.Center) {
+        ElevatedButton(
+            onClick = {
+              /** TODO : Start Activity */
+            },
+            modifier = Modifier.width(305.dp).height(48.dp),
+            colors = ButtonDefaults.buttonColors(containerColor = Color(0xFF0096CF))) {
+              Text(
+                  "Start",
+                  style =
+                      TextStyle(
+                          fontSize = 22.sp,
+                          lineHeight = 28.sp,
+                          fontWeight = FontWeight(400),
+                      ))
+            }
+      }
+}
+
+@Composable
+fun MiddleZone(activity: OutdoorActivity) {
+  Row {
+    DiffAndRating(activity)
+    Spacer(Modifier.width(170.dp))
+    ViewOnMapButton()
+  }
+}
+
+@Composable
+fun ViewOnMapButton() {
+  Column(modifier = Modifier.padding(vertical = 25.dp), horizontalAlignment = Alignment.End) {
+    ElevatedButton(
+        onClick = {
+          /** TODO : Go to map */
+        },
+        contentPadding = PaddingValues(all = 3.dp),
+        modifier = Modifier.width(130.dp).height(40.dp),
+        colors = ButtonDefaults.buttonColors(containerColor = Color(0xFF0096CF))) {
+          Text(
+              "View On Map",
+              style =
+                  TextStyle(
+                      fontSize = 16.sp,
+                      lineHeight = 24.sp,
+                      fontWeight = FontWeight(500),
+                      letterSpacing = 0.15.sp,
+                  ))
+        }
+  }
+}
+
+@Composable
+fun DiffAndRating(activity: OutdoorActivity) {
+  Column(modifier = Modifier.padding(vertical = 5.dp)) {
+    ElevatedDifficultyDisplay(diff = fetchDiffText(activity))
+    RatingDisplay(4.3)
+  }
+}
+
+@Composable
+fun RatingDisplay(rating: Double) {
+  Row(modifier = Modifier.padding(vertical = 30.dp)) {
+    Icon(
+        imageVector = Icons.Filled.Star,
+        contentDescription = "Rating Star",
+        tint = Color(0, 150, 207, 255),

```

Comment:
Here, you could either use the "yellow" hardcoded value of the star, or the blue/green of our app.